### ISSUE TITLE
✌️ 부스트캠프 멤버십 학습스프린트 회고

### ISSUE THREAD

**Issue by utterances-bot** [taenykim/taeny.dev#8]
# ✌️ 부스트캠프 멤버십 학습스프린트 회고

Learn in Public

[https://taeny.dev/boostcamp-membership-review](https://taeny.dev/boostcamp-membership-review)

**Comment by kakasoo** [comment]
아니 혼자만 다른 미션 하셨어요!? 퀄리티 너무 높은 거 아닙니까. ㅋㅋ 주어진 요구사항 외에도 이것저것 하시는 모습에 놀랐습니다. 저도 분발해야 겠네요!

**Comment by cyjo9603** [comment]
진짜 미션 퀄리티...... 너무 좋은것 같아요! 비공식 모임때 만나서 엄청 반가웠습니다 ㅎㅎ 글 보고 자극 많이 받아가요! 👍👍👍

**Comment by thdwlsgus0** [comment]
태은님.. 진짜 대단하십니다.. 존경합니다 ㅠㅠㅠㅠ

**Comment by taenykim** [comment]
@kakasoo 아이고ㅋㅋㅋㅋㅋ 과찬이십니다..ㅜㅠ 저도 3-4주차때 경수님이 많은 꿀팁들을 적극적으로 공유해주셔가지고 많이 배웠는데 저도 앞으로 적극적으로 공유하는 마인드를 가져야겠습니다!!ㅎㅎ 감사합니다!!

**Comment by taenykim** [comment]
@cyjo9603 저두 tech talk때도 그렇고, 마스터클래스 때도 best practice로 자주 보이셔서 뵙고 싶었는데 만나서 정말 반가웠습니다!!ㅎㅎ 감사합니다 찬영님~!

**Comment by taenykim** [comment]
@thdwlsgus0 헉! 아닙니다 ㅠㅠㅠㅠㅠ 감사해요 진현님!! ☺️

**Comment by enhakkore** [comment]
태은님 후기 너무 잘 봤어요ㅎㅎ 학습 스프린트에서 많이 성장하신것 같아요!!👍 저도 분발하겠습니다!!

**Comment by kyu9341** [comment]
역시 갓태은님!! 태은님 글 너무 좋아요! ㅎㅎ 👏
오프라인으로 만났을 때 너무 반가웠습니다~

**Comment by junsushin-dev** [comment]
역시 태은님 프론트 고수이시네요... 첫 미션때도 실력이 엄청나신 것은 알았지만 마지막 미션에는 더 성장하신 모습이 멋집니다!!

**Comment by taenykim** [comment]
@enhakkore 치현님 잘지내시죠!!!ㅋㅋㅋ 멘탈 흔들릴 때 따로 위로해주셔서 정말 감사했습니다..ㅜㅜㅜ 감사합니다~!ㅎㅎ

**Comment by taenykim** [comment]
@kyu9341 엇...ㅋㅋ 갓영언님.. 같은 그룹이 된 적은 없지만 실력이 좋으시다고 건너건너 소식이 다 들리던데요~~! ㅋㅋㅋ 저도 직접 만나서 너무 반가웠습니다! 감사합니다!!ㅎㅎ

**Comment by taenykim** [comment]
@junsushin-dev 헉 아닙니다..!  여담으루 준수님 싱글톤 패턴 발표 듣고서 바로 마지막 미션 때 적용해봤는데..ㅎ 항상 준수님께 많이 배우고 있습니다~!!! 좋은 말씀 감사해여~!! 😊

**Comment by seokju2ng** [comment]
태은님 모임에서는 유쾌하신 분으로 느꼈었는데, 능력자기까지 하셨네요....

**Comment by taenykim** [comment]
@seokju2ng 흑흑 아닙니다ㅠㅠㅠㅠㅠ 저도 석중님 덕분에 모임 때 너무 재밌었습니다 ㅋㅋㅋ 과분한 칭찬 넘무 감사합니다..!! 😊
